docs/learning/quantity/index.md

```diff
@@ -0,0 +1,225 @@
+# Quantity Formatting Package
+
+- [Quantity Formatting Package](#quantity-formatting-package)
+  - [Terms and Concepts](#terms-and-concepts)
+    - [Common Terms](#common-terms)
+    - [Concepts](#concepts)
+      - [Units Provider](#units-provider)
+      - [Unit Conversion](#unit-conversion)
+    - [Examples of Usage](#examples-of-usage)
+      - [Numeric Format](#numeric-format)
+      - [Composite Format](#composite-format)
+      - [Parsing Values](#parsing-values)
+      - [Mathematical Operation Parsing](#mathematical-operation-parsing)
+
+The __@itwin/core-quantity__ package contains classes for quantity formatting and parsing.
+For detailed API documentation, see our [iTwin.js reference documentation](https://www.itwinjs.org/reference/core-quantity/quantity/).
+
+If you're developing a frontend application that takes advantage of the core quantity APIs, also check out the [iTwin.js frontend quantity formatting learning section](../frontend/QuantityFormatting.md).
+
+## Terms and Concepts
+
+### Common Terms
+
+- [Unit]($quantity)/[UnitProps]($quantity) - A named unit of measure which can be located by its name or label. The definition of any unit is represented through its `UnitProps`.
```

Comment:
Do we need common terms here and in frontend/QuantityFormatting?

Reply:
yeah now that these are both learning pages, it's redundant. So I've moved everything over to quantity's index page surrounding common terms, and added links to this page from QuantityFormatting.md